app/src/main/java/com/sample/appconnectsample/RegistrationActivity.java

```diff
@@ -1,8 +1,17 @@
 package com.sample.appconnectsample;
 
 import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 
+import com.google.android.gms.appindexing.Action;
+import com.google.android.gms.appindexing.AppIndex;
+import com.google.android.gms.common.api.GoogleApiClient;
```

Comment:
These should be unused now that that autogenerated code is gone (same may be true for Uri and Bundle above)